securedrop/store.py

```diff
@@ -314,9 +329,13 @@ def save_file_submission(self,
                              filesystem_id: str,
                              count: int,
                              journalist_filename: str,
-                             filename: str,
-                             stream: 'BufferedIOBase') -> str:
-        sanitized_filename = secure_filename(filename)
+                             filename: typing.Optional[str],
+                             stream: 'IO[bytes]') -> str:
+
+        if filename is not None:
+            sanitized_filename = secure_filename(filename)
+        else:
+            sanitized_filename = secure_filename("unknown.file")
```

Comment:
You could simplify this by switching the argument's declaration to:
```
filename: str = "unknown.file",
```
Then no if/else would be needed.

Reply:
I kindof like it as is, I think @legoktm's previous point about readability is a good one.